Modules/DevTools/TestFramework/TestRunner/src/TestSuiteMgt.Codeunit.al

```diff
@@ -267,7 +267,6 @@ codeunit 130456 "Test Suite Mgt."
         TestRunnerGetMethods: Codeunit "Test Runner - Get Methods";
     begin
         BackupTestMethodLine.Copy(TestMethodLine);
-        TestMethodLine.Reset();
```

Comment:
Hi Luc,

This will not work. We have a bug already: If you put a filter to the individual test method and select UpdateTestMethods action, it will load all methods except that method. That method will disappear.

Also the problem is in the behavior, I would not expect that only filtered list is updated.

Could we do this implementation:
Check if there is a single codeunit, if yes, then update all
If there is multiple, prompt the user to select if they only want to update all (default) or selected? (Something similar to Run Tests action) 
Then we can keep this code as it is and add new path to update selected methods codeunit.

Reply:
I am totally fine with your suggestion. Thanx for the effort.